lgc/test/BuiltIns/cs-subgroupid.lgc

```diff
@@ -1,5 +1,5 @@
 ; NOTE: Assertions have been autogenerated by tool/update_llpc_test_checks.py UTC_ARGS: --tool lgc
-; RUN: lgc -mcpu=gfx802 - < %s | FileCheck --check-prefixes=WAVE64 %s
+; RUN: lgc -mcpu=gfx1010 - < %s | FileCheck --check-prefixes=WAVE64 %s
 ; RUN: lgc -mcpu=gfx1010 - < %s | FileCheck --check-prefixes=WAVE32 %s
```

Comment:
It doesn't make sense to have two identical run lines. I think you can just remove the gfx802 RUN line and remove the WAVE64 check lines as well. Realistically, CTS ought to be better at capturing problems here.

Reply:
Got it. Fixed.